service/history/historyEngine.go

```diff
@@ -580,6 +580,14 @@ Update_History_Loop:
 		var continueAsNewBuilder *mutableStateBuilder
 		hasDecisionScheduleActivityTask := false
 
+		if request.StickyAttributes == nil {
```

Comment:
Can we add a metric on requests with sticky tasklists

Reply:
added e.metricsClient.IncCounter(metrics.HistoryRespondDecisionTaskCompletedScope, metrics.CompleteDecisionWithStickyEnabledCounter)
and e.metricsClient.IncCounter(metrics.HistoryRespondDecisionTaskCompletedScope, metrics.CompleteDecisionWithStickyDisabledCounter)